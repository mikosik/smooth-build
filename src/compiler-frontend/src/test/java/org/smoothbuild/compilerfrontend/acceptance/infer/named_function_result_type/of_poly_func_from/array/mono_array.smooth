
myFunc(A a) = [7];