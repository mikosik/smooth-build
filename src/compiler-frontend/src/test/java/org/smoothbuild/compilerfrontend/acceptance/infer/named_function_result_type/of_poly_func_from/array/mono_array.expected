[Int] myFunc<A>(A a)
  = [7];
